android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Retailer.kt

```diff
@@ -45,27 +52,107 @@ class Retailer {
     }
 
     @OptIn(ExperimentalCoroutinesApi::class)
-    fun account(
-        client: AccountLinkingClient,
-        retailerId: Int,
-        email: String,
-        password: String,
-    ): CompletableDeferred<Boolean> {
+    fun account(call: PluginCall): CompletableDeferred<Boolean> {
+        val req = ReqRetailerAccount(call.data)
         val account = Account(
-            retailerId,
-            PasswordCredentials(email, password)
+            req.retailerId.value,
+            PasswordCredentials(req.username, req.password)
         )
         val isAccountLinked = CompletableDeferred<Boolean>()
         client.link(account)
             .addOnSuccessListener {
-                isAccountLinked.complete(it)
+                clientVerification(call){
+                    val rsp = RspRetailerAccount(req.username, req.retailerId)
+                    call.resolve(JSObject.fromJSONObject(rsp.toJson()))
+                    isAccountLinked.complete(it)
+                }
             }
             .addOnFailureListener {
                 isAccountLinked.completeExceptionally(it)
```

Comment:
Treat errors. Check email login method. Call the clientVerification method if necessary.